extensions/amp-list/0.1/amp-list.js

```diff
@@ -476,7 +483,11 @@ export class AmpList extends AMP.BaseElement {
    */
   addElementsToContainer_(elements, container) {
     elements.forEach(element => {
-      if (!element.hasAttribute('role')) {
+      if (
+        !element.hasAttribute('role') && 
+        // when 'single-item' is specified, do not add role to child element
```

Comment:
Nit: Might move this comment inside the condition check and explain why not setting the attribute.

`listitem` added to a single item is confusing for screenreaders.

Reply:
Thanks!  I have updated the comment to give a better explanation and placed it inside the condition check.